website/addons/base/__init__.py

```diff
@@ -76,13 +76,16 @@ def __init__(self, short_name, full_name, owners, categories,
         self.models = models
         self.settings_models = {}
 
-        if node_settings_model:
-            node_settings_model.config = self
-            self.settings_models['node'] = node_settings_model
+        from django.apps import apps
+        app = apps.get_app_config('addons_{}'.format(short_name))
```

Comment:
Just a heads up: I am currently working on removing the `AddonConfig` class, and consolidating all constants/configuration in the AddonAppConfig classes.